Courses/_lib/src/enrollment.js

```diff
@@ -182,8 +179,7 @@ window.intercomSettings = {
       form.onSuccess(function (values, _) {
         // FIXME: unsafe, accessToken can be undefined!
         enrollStudentInClass(values['FirstName'], values['LastName'], accessToken).done(function () {
-          record_event('training', trainingRegisterEvent)
-          gcSendEvent('online_training', 'register', trainingRegisterEventDetail)
```

Comment:
I removed this Google Analytics send event because we are already doing it in `record_event`.